README.md

```diff
@@ -1,2 +1,19 @@
-# label-supported-dropwizard-metrics
+# Label supported JMX metrics
 Provides an object name factory for Dropwizard that adds the support for metric labels.
+Labels are of great value for example on aggregating metrics by Prometheus.
+
+### Sample Usage
+
+```java
```

Comment:
Rewrite this section:

The client code for referring to a label metric is like this:
 * <pre>
 *  metricRegistry.counter(LabeledMetric.name("num_records").label("device_id", "1312").toString()).mark();
 * </pre>
 * Although is possible to have metrics without labels same as before.
 *
 * <p>Although to make it work, you should also introduce this name factory when starting the JMX reporter:
 * <pre>
 *  JmxReporter.forRegistry(metricRegistry)
 *             .createsObjectNamesWith(new LabelSupportedObjectNameFactory())
 *             .inDomain("my-metrics-domain")
 *             .build().start();
 * </pre>